### DISCUSSION TITLE
Please bring back old marketplace

### DISCUSSION THREAD

**Discussion by marverix** [community/community#126355] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Please bring back old marketplace! Not only the search auto-submit is horrible and interrupts me with writing (who designed it? has it been discussed with UX responsible for Web Accessibility?), but I cannot find anything. I cannot search only for actions. Tile has no information how much starts the action has. PLEASE! ROLLBACK!

**Comment by ReenigneArcher** [comment]
+1

I guess it looks nice, but functionality is useless. We need star count and ability to search for apps OR actions.

**Comment by WesleyAtanasio** [comment]
really
